gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,31 @@
+id: gigamon
+backend_only: true
+facets:
+  - facetType: list
+    groups:
+      - gigamon
+    name: Application Name
+    path: app_name
```

Comment:
What is this exactly? not seeing this in the samples and I'm wondering if it should be mapped to `service`?